python/service_bundle.py

```diff
@@ -120,6 +120,18 @@ def create_taskstatuses_file(self):
         else:
             self.write_file("service_v1_debug_taskStatuses.json", response.text)
 
+    @config.retry
+    def create_taskreservations_file(self):
```

Comment:
I'm not sure about the `task` prefix. The actual file does not have it, so this might be confusing...